apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/styles/starter-page-templates-editor.scss

```diff
@@ -476,3 +391,104 @@ body.admin-bar:not( .is-fullscreen-mode ) .page-template-modal-screen-overlay {
 	justify-content: space-around;
 	margin-top: 20px;
 }
+
+.block-iframe-preview {
+	position: absolute;
+	top: 0;
+	left: 0;
+	transform-origin: top left;
+	text-align: initial;
+	margin: 0;
+	overflow: visible;
+	min-height: auto;
+
+	width: 1440px;
+}
+
+.block-iframe-preview-body {
+	margin: 0;
+	padding: 0;
+	overflow-x: hidden;
+	overflow-y: auto;
+
+	> .block-editor,
+	> .block-editor > .edit-post-visual-editor {
+		padding: 0;
+		margin: 0;
+	}
+
+	&.not-selected {
+		 // Adjust template block title.
+		.wp-block:first-child {
+			margin-top: 0;
+			padding: 10px;
+		}
+	}
+
+	// Hide inserter/appender.
+	.block-list-appender,
+	.block-editor-inserter {
+		display: none !important;
+		visibility: hidden;
+		position: absolute;
+		left: -9999vw;
+	}
+
+	// Preview adjustments.
+	.editor-styles-wrapper {
+
+		// core/cover. The following styles are applied using inline-styles
+		.block-library-cover__resize-container {
+			position: relative;
+			user-select: auto;
+			width: auto;
+			height: auto;
+			min-height: 50px;
+			box-sizing: border-box;
+
+			.wp-block-cover {
+				height: auto;
+			}
+		}
+
+		// coblocks/gallery mock
+		.coblocks-gallery {
```

Comment:
Could we please not include override-CSS like this if it's normally generated by JS? Let's solve the root problem instead.

It's going to be hugely problematic for keeping this CSS updated together with CoBlocks plugin version updates. 

The JS also takes into account the viewport width, number, and size of images, so this CSS will need updating each time we change anything in the template or if we want to implement mobile previews, or just scale down the page in the browser window.

Reply:
Yeah, it's a good point. It will be a tricky one to tackle.